pkg/syncer/syncer.go

```diff
@@ -45,9 +45,10 @@ import (
 )
 
 const (
-	resyncPeriod       = 10 * time.Hour
-	SyncerNamespaceKey = "SYNCER_NAMESPACE"
-	syncerApplyManager = "syncer"
+	resyncPeriod        = 10 * time.Hour
+	SyncerNamespaceKey  = "SYNCER_NAMESPACE"
+	syncerApplyManager  = "syncer"
+	targetUIDAnnotation = "workload.kcp.dev/target.uid"
```

Comment:
(No action required) Maybe dashes are used instead of periods in the non-domain part of the key (e.g. target-uid)?

Reply:
I agree, I'll do it if we decide to pursue the work on that PR.